substrate/primitives/core/fuzz/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "sp-core-fuzz"
+version = "0.0.0"
+publish = false
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+lazy_static = "1.4.0"
+libfuzzer-sys = "0.4"
+regex = "1.10.2"
+
+sp-core = { path = ".." }
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
```

Comment:
Wait, its not part of the workspace? Will it have its own `Cargo.lock` then?

Reply:
@ggwpez: Yes.

Do you think it is a problem? I included fuzzer as a convenience tool intended to be executed locally.
What would be your advice here?